src/storage/GetProcessor.cpp

```diff
@@ -36,10 +36,14 @@ GetProcessor::asyncProcess(PartitionID part,
                            const std::vector<std::string>& keys) {
     folly::Promise<std::pair<PartitionID, kvstore::ResultCode>> promise;
     auto future = promise.getFuture();
+    std::vector<std::string> kvKeys;
+    std::transform(keys.begin(), keys.end(), std::back_inserter(kvKeys),
```

Comment:
We'd better reserve enough space before using kvKeys to avoid extra malloc.

Reply:
Good suggestion.